<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <widget class="QPushButton" name="btnTransladar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>71</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Transladar</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinDx">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>72</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinDy">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>72</width>
         <height>25</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
      <widget class="QComboBox" name="comboObjetos">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>151</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnEscalar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>120</y>
         <width>71</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Escalar</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinSx">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>120</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinSy">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>150</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>120</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>160</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnRotacionar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>200</y>
         <width>71</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Rotacionar</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spinAng">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>200</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>200</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>o</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="Meu_frame" name="Mundo">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <widget class="QPushButton" name="btnPanCima">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>460</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>CIMA</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPanBaixo">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>520</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>BAIXO</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPanEsquerda">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>490</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ESQ</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnPanDireita">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>490</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>DIR</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnZoomIn">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>490</y>
         <width>51</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Zoom +</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnZoomOut">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>520</y>
         <width>51</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Zoom -</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnGirarHorario">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>0</y>
         <width>71</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Horario</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnGirarAntiHorario">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>0</y>
         <width>71</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>AntiHorario</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Meu_frame</class>
   <extends>QFrame</extends>
   <header location="global">meu_frame.h</header>
   <container>1</container>
   <slots>
    <slot>muda()</slot>
    <slot>desenhar()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
